src/arcaflow_plugin_sdk/schema.py

```diff
@@ -2731,13 +2731,13 @@ def _insert_discriminator(
         discriminator field is moved to the zeroth index of the list of
         required fields in a data packet.
 
-        :param discriminated_object: A dictionary in the form of a json
-            fragment of an object's schema in the scope of this OneOf's union.
+        :param discriminated_object: A Python dict which represents the
+            relevant fragment of the scope's JSON definition.
         :param discriminator_val: The value that represents the given object in
             its OneOf's union.
-        :returns: A dictionary in the form of a json fragment of an object's
-            schema in the scope of this OneOf's union updated to reflect its
-            membership in this OneOf's union.
+        :returns: A Python dict which represents the relevant fragment of the
+            scope's JSON definition updated to reflect its membership in this
+            OneOf's union.
```

Comment:
Several "OneOf" references here that I don't think is actually a type. "discriminated union"??

Reply:
did it